### PR TITLE
Cross-link to quarkiverse wiki

### PR DESCRIPTION
**Pull request by holly-cummins**
I had some difficulty figuring out the link between writing an extension and the Quarkiverse hub, so I have cross-linked from the guide to the wiki.

### CODE REVIEW

**Review by HeroxHeruxum — APPROVED** [review]

File: docs/src/main/asciidoc/writing-extensions.adoc [hunk 1]
```diff
@@ -3189,15 +3189,27 @@ Then, at the end of your documentation, include the extensive documentation:
 
 Finally, generate the documentation and check it out.
 
+[[ecosystem]]
+== Ecosystem integration
+
+Some extensions may be private, and some may wish to be part of the
+broader Quarkus ecosystem, and available for community re-use.
+Inclusion in the Quarkiverse Hub] is a convenient mechanism for handing
```
Comment (gsmet) [inline docs/src/main/asciidoc/writing-extensions.adoc:3197]:
```suggestion
Inclusion in the Quarkiverse Hub is a convenient mechanism for handing
```

File: docs/src/main/asciidoc/writing-extensions.adoc [hunk 2]
```diff
@@ -3189,15 +3189,27 @@ Then, at the end of your documentation, include the extensive documentation:
...
+continuous testing and publication.
+The link:https://github.com/quarkiverse/quarkiverse/wiki#getting-an-extension-onboarded[Quarkiverse hub wiki] has instructions
```
Comment (gsmet) [inline docs/src/main/asciidoc/writing-extensions.adoc:3199]: Not sure if it should be capitalized but let's be consistent between above and this one.
```suggestion
The link:https://github.com/quarkiverse/quarkiverse/wiki#getting-an-extension-onboarded[Quarkiverse Hub wiki] has instructions
```

**Review by gsmet — COMMENTED** [review]
Added 2 small suggestions.

File: docs/src/main/asciidoc/writing-extensions.adoc (hunk 1 above)
Comment (gastaldi) [inline docs/src/main/asciidoc/writing-extensions.adoc:3197]:
```suggestion
Inclusion in the Quarkiverse Hub] is a convenient mechanism for handling
```

**Comment by gsmet** [comment]
@holly-cummins could you have a look at the suggestions so we merge that one? Thanks.

**Comment by holly-cummins** [comment]
Thanks for the fixes, @gastaldi , and thanks for the nudge, @gsmet! I apparently missed the github mail, and wondered why this was stuck. :)

**Comment by gsmet** [comment]
I squashed the commits and force pushed.

**Review by gsmet — APPROVED** [review]

**Pull request merged**